package.json

```diff
@@ -2,21 +2,26 @@
   "name": "@gnosis.pm/dex-js",
   "version": "0.0.1",
   "description": "",
+  "license": "MIT",
   "scripts": {
-    "sandbox": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon -r dotenv/config -r tsconfig-paths/register",
+    "build": "rimraf build && tsc",
     "test": "jest",
     "test-coverage": "jest --coverage && cat ./coverage/lcov.info | coveralls",
     "lint": "eslint src --ext .ts,.js",
     "lint:fix": "eslint --fix . --ext .ts,.js",
     "verify": "npm run lint && npm run test",
-    "build": "rimraf build && tsc",
-    "rpc": "ganache-cli -d --mnemonic 'candy maple cake sugar pudding cream honey rich smooth crumble sweet treat'"
+    "sandbox": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon -r dotenv/config -r tsconfig-paths/register"
```

Comment:
why `-r tsconfig-paths/register` when we don't use paths?

Reply:
It was used to fix the resolution, however at this point there's no YET a resolution problem, so I'll remove it for now until it appears again (or hopefully not)

Reply:
eec3721f24e8661715934acc832070b44db37640